File: web/Multa/multa.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

$(document).ready(
        function () {

            $.get('../ServletConsultarUsuarioIniciado', function (response) {
                console.log(response);
                             
                if (response=== 'null') {
                    console.log(response);
                    $(location).attr('href', "../Session/sesion.html");
                } else {


                }
            });

            $.get('/ProyectProfeco/api/multas/',
                    function (datos)
                    {
                        var infoJSON = datos;
                        console.log(datos);

                        console.log(infoJSON);
                        var infoJSONhtml;
                        for (var i = 0; i < infoJSON.length; i++) {
                            infoJSONhtml += '<tr> <td>' + infoJSON[i].idMulta + '</td> <td>' + infoJSON[i].idSupermercado + '</td> <td>' + infoJSON[i].tipoSancion + '</td> <td>' + infoJSON[i].numInconsistencias + '</td> <td> <button type="button" class="btn btn-secondary edit" data-toggle="modal" data-target="#ModalEliminar" onclick="MandarDatos(' + infoJSON[i].idMulta + ',`' + infoJSON[i].idSupermercado + '`,`' + infoJSON[i].TipoSancion + '`,`' + infoJSON[i].NumInconsistencias + '`)">Eliminar</button> </td></tr>';
                        }
                        $('#tbody').html(infoJSONhtml);
//                       |
                    }
            );



            MandarDatos = function (xid, xnombre, xdireccion) {


                $('#ModalEliminar').on('show.bs.modal', function (e) {
                    var id = xid;
                    var nombre = xnombre;
                    var direccion = xdireccion;
                    $('#eid').val(id);
                    $('#eNombre').val(nombre);
                    $('#eDireccion').val(direccion);
                });
            }

            $('#btnEliminar').click(
                    function (e) {
                        e.preventDefault();


                        var id = $('#eid').val();


                        $.ajax({
                            url: '/ProyectProfeco/api/multas/' + id,
                            method: 'delete',
                            contentType: "application/json",
                            Accept: "application/json",
                            success: function (response) {
                                alert("funciona bien");
                                $(location).attr('href', "multa.html");
                            },
                            error: function (error) {
                                alert("No funciona");
                            }
                        });
                    }

            );


        }

);

File: src/java/profeco/negocio/app/dto/Multa.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package profeco.negocio.app.dto;

/**
 *
 * @author Luis Barroso
 */
public class Multa {
    private int idMulta;
    private int idSupermercado; 
    private String TipoSancion;
    private int NumInconsistencias;
   

    public Multa() {
    }

    public Multa(int idMulta, int idSupermercado, int NumInconsistencias, String TipoSancion) {
        this.idMulta = idMulta;
        this.idSupermercado = idSupermercado;
        this.NumInconsistencias = NumInconsistencias;
        this.TipoSancion = TipoSancion;
        
    }

    public int getIdMulta() {
        return idMulta;
    }

    public int getIdSupermercado() {
        return idSupermercado;
    }

    public int getNumInconsistencias() {
        return NumInconsistencias;
    }

    public String getTipoSancion() {
        return TipoSancion;
    }

   
    public void setIdMulta(int idMulta) {
        this.idMulta = idMulta;
    }

    public void setIdSupermercado(int idSupermercado) {
        this.idSupermercado = idSupermercado;
    }

    public void setNumInconsistencias(int NumInconsistencias) {
        this.NumInconsistencias = NumInconsistencias;
    }

    public void setTipoSancion(String TipoSancion) {
        this.TipoSancion = TipoSancion;
    }

    
    
}
